x-pack/plugins/viz_editor/common/select_builder.ts

```diff
@@ -19,17 +19,30 @@ const operationTemplate: OperationTemplate = {
     operation: 'date_histogram',
     argument: {
       field: '',
-      interval: 'auto',
+      interval: 'd',
```

Comment:
I think we need to provide a timerange to esaggs for auto to work correctly. We should think about adding the filter and query bar next week or so

Reply:
Makes sense